sources/Outputs/src/Par.cpp

```diff
@@ -475,5 +486,51 @@ Par::updatePropParameters(boost::shared_ptr<parameters::ParametersSet> set) {
   set->addReference(helper::buildReference("generator_QRef0Pu", "targetQ_pu", "DOUBLE"));
   set->addReference(helper::buildReference("generator_QPercent", "qMax_pu", "DOUBLE"));
 }
+
+boost::shared_ptr<parameters::ParametersSet>
+Par::writeStaticVarCompensator(const inputs::StaticVarCompensator& svarc) {
+  auto set = boost::shared_ptr<parameters::ParametersSet>(new parameters::ParametersSet(svarc.id));
+
+  set->addMacroParSet(boost::make_shared<parameters::MacroParSet>(macroParameterSetStaticCompensator_));
+
+  double value = computeBPU(svarc.bMax, svarc.VNom);
+  set->addParameter(helper::buildParameter("SVarC_BMaxPu", value));
+
+  value = computeBPU(svarc.bMin, svarc.VNom);
+  set->addParameter(helper::buildParameter("SVarC_BMinPu", value));
+
+  value = computeBPU(svarc.b0, svarc.VNom);
+  set->addParameter(helper::buildParameter("SVarC_BShuntPu", value));
+
+  value = svarc.slope * Sb_ / svarc.VNom;
+  set->addParameter(helper::buildParameter("SVarC_LambdaPu", value));
+
+  set->addParameter(helper::buildParameter("SVarC_UNom", svarc.VNom));
+  set->addParameter(helper::buildParameter("SVarC_URefDown", svarc.USetPointMin));
+  set->addParameter(helper::buildParameter("SVarC_URefUp", svarc.USetPointMax));
+  set->addParameter(helper::buildParameter("SVarC_UThresholdDown", svarc.UMinActivation));
+  set->addParameter(helper::buildParameter("SVarC_UThresholdUp", svarc.UMaxActivation));
+  set->addParameter(helper::buildParameter("URef_ValueIn", svarc.voltageSetPoint));
+
+  return set;
+}
+
+boost::shared_ptr<parameters::MacroParameterSet>
+Par::writeMacroParameterSetStaticVarCompensators() {
+  auto macro = boost::make_shared<parameters::MacroParameterSet>(macroParameterSetStaticCompensator_);
+
+  macro->addReference(helper::buildReference("SVarC_Mode0", "regulatingMode", "INT"));
+  macro->addReference(helper::buildReference("SVarC_P0Pu", "p_pu", "DOUBLE"));
+  macro->addReference(helper::buildReference("SVarC_Q0Pu", "q_pu", "DOUBLE"));
+  macro->addReference(helper::buildReference("SVarC_U0Pu", "v_pu", "DOUBLE"));
+  macro->addReference(helper::buildReference("SVarC_UPhase0", "v_pu", "DOUBLE"));
```

Comment:
here it is a phase, not a voltage (angle_pu)

Reply:
OK